ui-performance/index.md

```diff
@@ -345,17 +372,29 @@ When you encounter calls to `saveLayer`, ask yourself these questions:
 
 * Does the app need this effect?
 * Can any of these calls be eliminated?
+* Can I apply the same effect to an individual element instead of a group?
 
 #### Checking for non-cached images
 
-Image I/O, such as passing images from host memory to device memory,
-can be expensive. Where possible, you want to cache images.
+One of the most expensive operations, from a resource perspective, is
+rendering a texture using an image file. First, the compressed image is
+fetched from persistent storage (disk, for example).
```

Comment:
Lets not mention a disk. I don't think those have been a thing on a phone :D

Reply:
Done